doc-content/enterprise-only/installation/clustering-bc-install-proc.adoc

```diff
@@ -9,7 +9,7 @@ After you have configured a {EAP} cluster and repackaged {CENTRAL}, you must sta
 * The repackaged `{URL_COMPONENT_CENTRAL}.war` file is available.
 
 .Procedure
-. Add {EAP} management users to the master node of the cluster (where you configured the `domain.xml` file) as described in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/{EAP_VERSION}/html-single/configuration_guide/[_Red Hat JBoss EAP 7.3 Configuration Guide_].
+. Add {EAP} management users to the master node of the cluster (where you configured the `domain.xml` file) as described in the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/{EAP_VERSION}/html-single/configuration_guide/[_Red Hat JBoss EAP 7.4 Configuration Guide_].
```

Comment:
I searched for this doc and it doesnt appear to be used in any assembly. Double check by searching for clustering-bc-install-proc.adoc  in the repo. If you don't find it in an include statement, then delete.

Reply:
Deleted.